android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/survey/SurveyScreen.kt

```diff
@@ -35,79 +27,77 @@ fun SurveyScreen(
     }
 
     SurveyScreenContent(
+        questions = listOf(
```

Comment:
I think we should add `// Todo` for future implementation.

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/68/commits/2e59021b21829a05a5491334409734a6ea400069